### DISCUSSION TITLE
Specifying subscription in Azure scan

### DISCUSSION THREAD

**Discussion by njmulsqb** [prowler-cloud/prowler#2089] [Q&A] [answered]
Hi,

I see that by default prowler scans all the subscriptions on azure, is there any way to specify only one subscription to scan with prowler?

**Comment by jfagoagas** [comment, marked as answer]
Hi @njmulsqb, you can scan one subscription using the `--subscription-ids` argument this way:
```
prowler azure --subscription-ids SUBSCRIPTION_ID
```

Thank you for using Prowler!

**Reply by toniblyx** [reply]
more info here https://docs.prowler.cloud/en/latest/tutorials/azure/subscriptions/
